spk/sshfs-fuse/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = sshfs-fuse
+SPK_VERS = 2.5
+SPK_REV = 1
+SPK_ICON = src/sshfs-fuse.png
+
+DEPENDS = cross/$(SPK_NAME) cross/fuse cross/glib cross/zlib cross/libffi
```

Comment:
For next time, entries should be alphabetical, with `cross/$(SPK_NAME)` last.
However, in this case, you might realize that all the entries here are dependencies of the cross package `sshfs-fuse`, but are not dependencies of the SPK package itself. You only need `DEPENDS = cross/$(SPK_NAME)` here.